app/frontend/src/details.js

```diff
@@ -18,9 +17,12 @@ function addDynamicSummaryTextForExpandedAndCollapsedDetailsTag(detailsElement)
     }
 }
 
-
```

Comment:
Jquery is a massive, insecure, slow way to do this.  We should probably install a modern JS framework and let it handle all of this.  I'm most experienced with [Vue](https://vue.js), and [React](https://react.js) is the mode widely used, but there are several others.  Won't block the PR on this, but we should discuss and get something in the roadmap to review and decide.